### ISSUE TITLE
Bug in AceRange or user error?

### ISSUE THREAD

**Issue by Ueland** [daveho/AceGWT#20]
Hello,

I have been experimenting a bit with the AceRange object, and tried to create a AceRange object via a JSNI function:

```javascript
var range = $entry(@edu.ycp.cs.dh.acegwt.client.ace.AceRange::create(IIII)(0, 0, 10, 100));
```

This throw the exception: "TypeError: $wnd.require is not a function"

I noticed that AceRange.java:47 has the following line:

```javascript
 var Range = $wnd.require('ace/range').Range;
```

Should not that rather be this: ?
```javascript
 var Range = $wnd.ace.require('ace/range').Range;
```

Doing the following in my on JNDI function appears to work:
```javascript
var Range = $wnd.ace.require('ace/range').Range;
          var editor = this.@edu.ycp.cs.dh.acegwt.client.ace.AceEditor::editor;
          var range = new Range(n, n, n, n);
          editor.getSession().addMarker(range,"foo", "foo", true);

```

Or is it just a matter of user error on my behalf? :)

**Comment by aranega** [comment]
Hi,

Indeed, you're right, I made a mistake :), it should be `$wnd.ace.require...`. I fixed this bug in local but totally forgot to send a pull request about it (I added other features that require polishing). This issue is also present in `AceEditor::setModeByName`. 
Actually, if I'm not wrong, this error occurs only if you use other library that redefine `require` (I had the same issue with dojo).

**Comment by Ueland** [comment]
That`s correct, but no worries, just had to confirm my theory that i was not doing it all wrong here :)
